.claude/skills/roughcut/templates/apply_recipe.py

```
@@ -0,0 +1,335 @@
+        for marker in clip.get("markers", []):
+            self.counts["markers"][1] += 1
+            frame = int(round(marker["at"] * self.frame_rate))
+            custom_data = f"buttercut:{marker['name']}"
+            # Make idempotent: a previous apply may have placed a buttercut
+            # marker at this same frame. Clear it before re-adding so the
+            # script can be run repeatedly.
+            try:
+                if hasattr(item, "DeleteMarkerByCustomData"):
+                    item.DeleteMarkerByCustomData(custom_data)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)